st/clitests/awsiam_spec.py

```diff
@@ -50,6 +50,16 @@ def get_policy_list_count(raw_aws_cli_output):
             continue
     return count
 
+def get_attached_user_policy_list_count(raw_aws_cli_output, result_prefix):
+    raw_lines = raw_aws_cli_output.split('\n')
+    count = 0
+    for _, item in enumerate(raw_lines):
+        if (item.startswith(result_prefix)):
```

Comment:
Just keep 1 method as its similar to above just generic one.
plz keep only this one and change references of above similar method.

Reply:
Done.